developers.diem.com/docs/core/run-local-network.md

```diff
@@ -5,7 +5,7 @@ title: Run a Local Network
 
 You can run a local test validator network to test and debug services you are developing for the Diem Blockchain and to build and test Move module code changes. This local test network is not part of the Diem Ecosystem and is only meant for testing and development purposes.
 
-You can use the Diem CLI command dev to compile, publish, and execute Move Intermediate Representation (IR) programs on your local test validator network.
+You can also use the Diem CLI command dev to compile, publish, and execute Move programs on your local test validator network.
 
 >
 >Note: Your local test network of validator nodes will not be connected to the testnet, and will not be using the actual validator nodes on the Diem Blockchain.
```

Comment:
`Note: Your local test network will not be connected to testnet, or mainnet of the Diem Blockchain.`